docs/src/main/sphinx/installation/cli.rst

```diff
@@ -29,6 +29,22 @@ The CLI uses the HTTP protocol and the
 :doc:`Trino client REST API </develop/client-protocol>` to communicate
 with Trino.
 
+TLS/HTTPS
+---------
+
+Typically a Trino server is available with a HTTPS URL, so all network traffic
+from the CLI uses TLS. This :doc:`TLS configuration </security/tls>` is common,
+since it is a requirement for any authentication.
+
+Connecting to the server simply requires you to use the HTTPS URL:
+
+.. code-block:: text
+
+    ./trino --server https://trino.example.com
+
```

Comment:
Remove "simply" - I'd recommend "Use the HTTPS URL to connect to the server:"

Reply:
Updated